src/artemis/devices/s4_slit_gaps.py

```diff
@@ -0,0 +1,7 @@
+from ophyd import Component, Device, EpicsSignal
+
+
+class S4SlitGaps(Device):
+
+    s4xgap: EpicsSignal = Component(EpicsSignal, "S4XGAP")
+    s4ygap: EpicsSignal = Component(EpicsSignal, "S4YGAP")
```

Comment:
Must: By changing the PV from `"XGAP"` to `"S4XGAP"` this is effectively telling ophyd to look for PVs called `BL03I-AL-SLITS-04:S4XGAP`, which I don't think exist:
```
[ffv81422@ws455 tickit-devices]$ caget BL03I-AL-SLITS-04:S4XGAP
Channel connect timed out: 'BL03I-AL-SLITS-04:S4XGAP' not found.
[ffv81422@ws455 tickit-devices]$ caget BL03I-AL-SLITS-04:XGAP
BL03I-AL-SLITS-04:XGAP         0.5
```

Reply:
Nit: I don't think I would change the `xgap`/`ygap` here necessarily `s4_slits.xgap` reads a bit better to me than `s4_slits.s4xgap` as it seems needless repetition.